src/graphql/schema/connection/createConnectionGqlField.ts

```diff
@@ -141,6 +142,7 @@ export function _createConnectionGqlType <TInput, TItemValue>(
   return new GraphQLObjectType({
     name: formatName.type(`${paginator.name}-connection`),
     description: `A connection to a list of ${scrib.type(gqlType)} values.`,
+    relatedGqlType: gqlType,
```

Comment:
It was the easiest way to concretely determine whether to keep travelling down the AST looking for related nodes; without this we either over-select (making us fetch fields we don't need) or we have to do duck typing by looking for edges/nodes and inferring from there (which makes me uncomfortable); or placing constraints on the naming patterns so if it's looking for a Team then it'd know to look for TeamConnection/TeamEdge/etc. I suggest we keep this unless you know a better way.